packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/inspection/inspect.ts

```diff
@@ -45,12 +50,13 @@ const makeInput = async (flags: any, args: string[]): Promise<ContractExpectedIn
       recommendedGasPriceMicro: info.billing.recommendedGasPriceMicro,
       transmissionPaymentGjuels: info.billing.transmissionPaymentGjuels,
     },
+    offchainConfig: getOffchainConfigInput(rdd, contract),
   }
 }
 
 const makeInspectionData = () => async (input: ContractExpectedInfo): Promise<ContractExpectedInfo> => input
 
-const makeOnchainData = (query: Query) => async (
+const makeOnchainData = (query: Query, search: Search) => async (
```

Comment:
Anyone reading this first question will be what's the difference between a `Query` and a `Search` 🤔

Reply:
Why don't we have a `provider` as an input param and then do:

- `provider.wasm.contractQuery({...})`
- `provider.tx.search({...})`

It would make the code a lot more understandable IMO.

Reply:
With provider works like a charm